src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -125,6 +136,12 @@ public Conversation getConversationWithID(UUID id) {
 	/** Sets the List of Conversations stored by this ConversationStore. */
 	public void setConversations(List<Conversation> conversations) {
 		this.conversations = conversations;
+		this.publicConversations = new ArrayList<Conversation>();
+		for (Conversation conversation : this.conversations) {
+			if (conversation.getIsPublic()) {
+				publicConversations.add(conversation);
+			}
+		}
 	}
   
```

Comment:
This file looks great; however, I was a little confused on why the "conversations" list is still kept. Is there a reason? We could use this list to include all conversations (group or public).

Reply:
I may not need it in the end, but I've kept it for now to keep track of all of the conversations, so group and public. Right now the conversation page only shows the public conversations, which is why I only use the publicConversations variable, but next I will also add the group/private conversations, so I think I'll need the conversations list there.